frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -0,0 +1,253 @@
+
+const pollIntervalOptions = ['5s', '15s', '30s', '1m', '2m', '5m', '15m', '30m'];
+const defaultPollInterval = '30s';
+
+// TODO: Dynamically load the list of dashboards
+const boards = [
+  'k8s-resources-cluster',
+  'k8s-resources-namespace',
+  'cluster-total',
+  'node-cluster-rsrc-use',
+];
+const DashboardDropdown: React.FC<{ onChange: (string) => void }> = ({ onChange }) => {
+  const items = _.zipObject(boards, boards);
+  return <Dropdown items={items} onChange={onChange} selectedKey={boards[0]} />;
+};
+
+const DurationDropdown: React.FC<DurationDropdownProps> = ({ items, onChange, selected }) => (
+  <Dropdown
+    buttonClassName="monitoring-dashboards-dropdown"
+    items={_.zipObject(items, items)}
+    onChange={(v: string) => onChange(parsePrometheusDuration(v))}
+    selectedKey={selected}
+  />
+);
+
+const Panel: React.FC<PanelProps> = ({ panel, pollInterval, timespan }) => {
+  const queries = _.map(panel.targets, 'expr').map(evaluateTemplate);
+  if (!queries.length) {
+    return null;
+  }
+
+  // If panel doesn't specify a span, try to get it from the gridPos or default to full width
+  let colSpan = panel.span;
+  if (!_.isNumber(colSpan)) {
+    colSpan = _.has(panel, 'gridPos.w') ? panel.gridPos.w / 2 : 12;
+  }
+
+  return (
+    <div className={`col-xs-${colSpan}`}>
+      <DashboardCard className="monitoring-dashboards-panel">
+        <DashboardCardHeader>
+          <DashboardCardTitle>{panel.title}</DashboardCardTitle>
+        </DashboardCardHeader>
+        <DashboardCardBody>
+          {panel.type === 'grafana-piechart-panel' && <BarChart query={queries[0]} />}
+          {panel.type === 'graph' && (
+            <Graph pollInterval={pollInterval} queries={queries} timespan={timespan} />
+          )}
+          {panel.type === 'row' && !_.isEmpty(panel.panels) && (
+            <div className="row">
+              {_.map(panel.panels, (p) => (
+                <Panel key={p.id} panel={p} pollInterval={pollInterval} timespan={timespan} />
+              ))}
+            </div>
+          )}
+          {panel.type === 'singlestat' && (
+            <SingleStat
+              decimals={panel.decimals}
+              format={panel.format}
+              pollInterval={pollInterval}
+              postfix={panel.postfix}
+              prefix={panel.prefix}
+              query={queries[0]}
+              units={panel.units}
+            />
+          )}
+          {panel.type === 'table' && <Table pollInterval={pollInterval} queries={queries} />}
+        </DashboardCardBody>
+      </DashboardCard>
+    </div>
+  );
+};
+
+const Board: React.FC<{ board: string; pollInterval: number; timespan: number }> = ({
+  board,
+  pollInterval,
+  timespan,
+}) => {
+  const [data, setData] = React.useState();
+  const [error, setError] = React.useState();
+
+  React.useEffect(() => {
+    const path = `${k8sBasePath}/api/v1/namespaces/openshift-monitoring/configmaps/grafana-dashboard-${board}`;
```

Comment:
Ok great, can take a look at the configmaps from here.

This is where we need to have the discussion about ConfigMap vs CRD and the RBAC ramifications, as users may not be able to access these resources.